platform/core.network/src/org/netbeans/core/network/proxy/NbProxySelector.java

```diff
@@ -145,7 +145,7 @@ public NbProxySelector() {
             case ProxySettings.AUTO_DETECT_PAC:
                 if (useSystemProxies ()) {
                     if (original != null) {
-                        res = original.select(uri);                   
+                        res = new ArrayList<>(original.select(uri));
```

Comment:
Minimal nitpick: The other branches of the conditional add their result to the list, that is initialized in line 66. Would it be an option to use the same approach here? If not, this is ok, just indicate so.

Reply:
Yes, it makes sense: now all uses of `res` variable just add items to it.